Security.sln

```diff
@@ -57,6 +57,8 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Owin.Security.Int
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "OpenIdConnect.AzureAdSample", "samples\OpenIdConnect.AzureAdSample\OpenIdConnect.AzureAdSample.xproj", "{3A7AD414-EBDE-4F92-B307-4E8F19B6117E}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.AspNetCore.Cookies.ChunkingCookieManager.Sources", "src\Microsoft.AspNetCore.Cookies.ChunkingCookieManager.Sources\Microsoft.AspNetCore.Cookies.ChunkingCookieManager.Sources.xproj", "{B6297507-EECF-4114-9B38-6CCE7F9C3EB5}"
```

Comment:
I feel like we should drop 'cookies' from the name. `Microsoft.AspNetCore.Cookies` isn't a namespace we ship anywhere